apps/dashboard/app/views/scripts/editable_form_fields/_editable_number.html.erb

```diff
@@ -8,13 +8,24 @@
 
   current_min = attrib.field_options[:min]
   current_max = attrib.field_options[:max]
+
+  fixed_id = "#{field_id}_fixed"
+  fixed_name = "#{form.object_name}[#{attrib.id}_fixed]"
 -%>
 
 <div class="editable-form-field">
-  <%=  create_widget(form, attrib, format: format) %>
+  <%=  create_widget(form, attrib, format: format, hide_fixed: false) %>
 
   <div class="d-none edit-group mb-3">
 
+    <div class="list-group col-md-4">
+      <div class="list-group-item mb-3">
+        <input type="checkbox" id="<%= fixed_id %>" name="<%= fixed_name %>" value="true" <%=  fixed ? 'checked' : '' %>>
+        <label class="form-check-label" for="<%= fixed_id %>"><%= t('dashboard.jobs_scripts_static_field') %></label>
+      </div>
+    </div>
```

Comment:
Since this is duplicate - can we make a partial for the same?

Reply:
Fixed.